com_connect.php

```diff
@@ -1439,7 +1442,7 @@ function com_connect_radio($atts)
 
     return ($labelStr && $label_position === 'before' ? $labelStr . $break : '') .
         '<input type="radio"'. $classStr . ($attr ? ' ' . implode(' ', $attr) : '') .
-            ( $is_checked ? ' checked' . (($doctype === 'xhtml') ? '="checked"' : ''). ' />' : ' />') .
+            ( $is_checked ? ' checked' . (($doctype === 'xhtml') ? '="checked"' : '') : '') . (($doctype === 'xhtml') ? ' />' : '>') .
```

Comment:
Not sure about this line. I might be reading it wrong but there seems to be an additional `: '') ` that's snuck in and I can't see the corresponding ternary `?`. What am I missing?

Reply:
Nope, ignore me. Spotted it.